.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           project: qa
 
       - samvera/engine_cart_generate:
-          cache_key: v6-internal-test-app-{{ checksum "qa.gemspec" }}-{{ checksum "spec/test_app_templates/lib/generators/test_app_generator.rb" }}-{{ checksum "lib/generators/qa/install/install_generator.rb" }}-<< parameters.rails_version >>-<< parameters.ruby_version >>
+          cache_key: v7-internal-test-app-{{ checksum "qa.gemspec" }}-{{ checksum "spec/test_app_templates/lib/generators/test_app_generator.rb" }}-{{ checksum "lib/generators/qa/install/install_generator.rb" }}-<< parameters.rails_version >>-<< parameters.ruby_version >>
```

Comment:
Any time a spec fixture config changes, the cache needs to regen which creates a new test app with the updated configs.